promql/promqltest/testdata/functions.test

```diff
@@ -316,6 +321,17 @@ eval instant at 10m predict_linear(testcounter_reset_middle_total[55m] @ 3000, 3
 eval instant at 70m predict_linear(testcounter_reset_middle_total[55m] @ 3000, 3600)
 	{} 89.54545454545455
 
+# predict_linear should ignore histogram with info annotation.
```

Comment:
Let's also test the pure histogram case (which should result in an empty result without annotation).